packages/react-scripts/template/src/serviceWorker.js

```diff
@@ -63,8 +63,8 @@ function registerValidSW(swUrl, config) {
         if (installingWorker == null) {
           return;
         }
-        installingWorker.onstatechange = () => {
-          if (installingWorker.state === 'installed') {
+        installingWorker.onstatechange = stateChangeEvent => {
```

Comment:
This change has to be made in the typescript template as well.

Reply:
Is the typescript template maintain in this repository?

Reply:
Yes, we now support TypeScript and do not recommend users use `react-scripts-ts` anymore.

Reply:
After some more testing, I was able to get this working without the changes I made. I have reverted the changes.